doc/tech_dev_checklist.rst

```diff
@@ -386,7 +386,7 @@ The following messages are excluded by default: I0020, I0021, W0704. (It is not
 these stand for. They are not documented in Pylint.)
 
 The following messages are excluded by default but are activated in our case (related to Python
-3): E1601, E1602, E1603, E1604, E1605, E1606, E1607, E1608, W1601, W1602, W1603, W1604,
+3): E1136, E1601, E1602, E1603, E1604, E1605, E1606, E1607, E1608, W0621, W1601, W1602, W1603, W1604,
```

Comment:
This is not fixed in the correct please. (These exceptions should be listed somewhat lower.)